src/Mod/Arch/importIFClegacy.py

```diff
@@ -1015,15 +1015,17 @@ def export(exportList,filename):
         # setting the IFC type
         if hasattr(obj,"Role"):
             ifctype = obj.Role.replace(" ","")
-        else:
-            ifctype = otype
-        if ifctype == "Foundation":
+        elif otype == "Foundation":
             ifctype = "Footing"
-        elif ifctype == "Rebar":
+        elif otype == "Rebar":
             ifctype = "ReinforcingBar"
-        elif ifctype in ["Part","Undefined"]:
+        elif otype == "Undefined":
             ifctype = "BuildingElementProxy"
-            
+        elif otype.split("::")[0] == "Part":
```

Comment:
I thought to do it like this @vocx-fc . What do you think? The other possibility that came to my mind is to have a function `get_object_workbench(obj)`, but that's so randomly used and would have worked the same way...

Reply:
Looks ugly to me, you can use `startswith("Part::")`.

Reply:
ok! that looks better

Reply:
done!